sdk/devtestlabs/Azure.ResourceManager.DevTestLabs/tests/Scenario/DevTestLabTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.DevTestLabs.Models;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.DevTestLabs.Tests
+{
+    internal class DevTestLabTests : DevTestLabsManagementTestBase
+    {
+        private DevTestLabCollection _devTestLabCollections = null;
+        public DevTestLabTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [SetUp]
+        public async Task SetUp()
+        {
+            TestResourceGroup = await CreateResourceGroup();
+            _devTestLabCollections = TestResourceGroup.GetDevTestLabs();
+        }
+
+        [RecordedTest]
+        public async Task CreateOrUpdateExistGetGetAllDelete()
+        {
+            // CreateOrUpdate
+            string labName = Recording.GenerateAssetName("lab");
+            var lab = await CreateDevTestLab(TestResourceGroup, labName);
```

Comment:
Why not like the other tests creating devTestLab in the setup?
Then, `CreateDevTestLab` can be moved to set up of the base class.